.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{ 
+	"name": "Rust",
+        "image": "mcr.microsoft.com/devcontainers/rust:1-bullseye",
+	"features": {
+                "ghcr.io/devcontainers/features/python:1": {
+                        "installTools": true,
+                        "version": "os-provided"
+                }
+        },
+        "onCreateCommand": "cargo --version && rustc --version"
```

Comment:
This is unnecessary. (It is ok to have it, but there is no need to have it.)

Reply:
It kind it triggers an update(?) of cargo and rustc. I think it's therefore good to have...

Reply:
Good point. I suppose Cargo reads the `rust-toolchain.toml` file and change the toolchain at the first time running `cargo` command.